es6-persistence/src/main/java/com/netflix/conductor/es6/dao/index/ElasticSearchRestDAOV6.java

```diff
@@ -130,12 +126,15 @@ public class ElasticSearchRestDAOV6 extends ElasticSearchBaseDAO implements Inde
     private final long asyncBufferFlushTimeout;
     private final ElasticSearchProperties properties;
 
+    private final RetryTemplate retryTemplate;
+
     static {
         SIMPLE_DATE_FORMAT.setTimeZone(GMT);
     }
 
     public ElasticSearchRestDAOV6(
             RestClientBuilder restClientBuilder,
+            RetryTemplate retryTemplate,
```

Comment:
Should this have a `@Qualifier`?

Reply:
no, the bean creation happens in ElasticsearchV6Configuration and the Qualifier is used there.